control/bdalg.py

```diff
@@ -280,7 +281,10 @@ def append(*sys):
     >>> sys = append(sys1, sys2)
 
     """
-    s1 = sys[0]
+    if not isinstance(sys[0], StateSpace):
```

Comment:
With the current code at the top of the file, this is checking to see if `sys[0]` is a `sp.signal.ltisys.StateSpace` object, not a `control.StateSpace` object.  It should still work, but basically is almost always getting called (?).